examples/inverse-calculations.py

```diff
@@ -25,32 +26,55 @@ def fan_efficacy(seer):
         return resdx.u(0.25,'W/(cu_ft/min)') + (resdx.u(0.18,'W/(cu_ft/min)') - resdx. u(0.25,'W/(cu_ft/min)'))/2.0 * (seer - 14.0) 
 
 
-def C_d(seer):
+def C_d_cooling(seer):
     if seer <= 12:
-        return 0.25
+        return 0.2
     elif seer >= 13:
         return 0.1
     else:
-        return 0.25 + (0.1 - 0.25)*(seer - 12.0)
+        return 0.2 + (0.1 - 0.2)*(seer - 12.0)
+
+# Heating
+# def hspf(seer): # Linear model fitted (R² = 0.994) based on data of the histrory of federal minimums (https://www.eia.gov/todayinenergy/detail.php?id=40232#).
+    # return 0.3526 * seer + 3.2627
+
+def fan_efficacy_heating(hspf):
+    if hspf <= 8.2:
+        return resdx.u(0.25,'W/(cu_ft/min)')
+    elif hspf >= 8.9:
+        return resdx.u(0.18,'W/(cu_ft/min)')
+    else:
+        return resdx.u(0.25,'W/(cu_ft/min)') + (resdx.u(0.18,'W/(cu_ft/min)') - resdx. u(0.25,'W/(cu_ft/min)'))/0.7 * (hspf - 8.2) 
+
+
+def C_d_heating(hspf):
+    if hspf <= 7.5:
+        return 0.2
+    elif hspf >= 7.85:
+        return 0.1
+    else:
+        return 0.2 + (0.1 - 0.2)*(hspf - 7.5)/0.35
+
 
 #%% Cooling inverse calculations with constraints
 seer_range = np.arange(6,26.5,0.1)
 cops = []
 for seer in seer_range:
-    root_fn = lambda x : resdx.DXUnit(gross_cooling_cop_rated = [x], fan_eff_cooling_rated = [fan_efficacy(seer)], c_d_cooling=C_d(seer)).seer() - seer
+    root_fn = lambda x : resdx.DXUnit(gross_cooling_cop_rated = [x], fan_eff_cooling_rated = [fan_efficacy_cooling(seer)], c_d_cooling=C_d_cooling(seer)).seer() - seer
     cop, solution = optimize.newton(root_fn,seer/3.33, full_output = True)
-    print(f"seer = {seer:.1f}, cop = {cop:.2f}, fan_eff = {resdx.convert(fan_efficacy(seer),'W/(m**3/s)','W/(cu_ft/min)'):.3f}, C_d = {C_d(seer):.3f}, converged = {solution.converged}, iter = {solution.iterations}")
+    print(f"seer = {seer:.1f}, cop = {cop:.2f}, fan_eff = {resdx.convert(fan_efficacy_cooling(seer),'W/(m**3/s)','W/(cu_ft/min)'):.3f}, C_d = {C_d_cooling(seer):.3f}, converged = {solution.converged}, iter = {solution.iterations}")
     cops.append(cop)
 
-plot(seer_range, cops, "SEER", "Gross COP")
-
+plot(seer_range, cops, "SEER", "Gross COP95 (fan not included)")
```

Comment:
I would suggest the axis title: "Gross COP (at 'A' conditions)"

Saying fan not included is redundant (though helpful for people that confuse gross/net).